frontend/app/pages/index.vue

```diff
@@ -23,12 +23,12 @@ useSeoMeta({
       <section class="my-8">
         <SectionInfoTiles />
       </section>
-      <section id="section-how-to-participate" class="my-8">
-        <LazySectionHowToParticipate hydrate-on-interaction />
-      </section>
-      <section id="sponsors" class="my-8">
-        <LazySectionSponsors hydrate-never />
-      </section>
+<!--      <section id="section-how-to-participate" class="my-8">-->
```

Comment:
You can remove this code. The question is should you keep those components... And I am unsure what the answer should be, but I think you can remove them because how-to-participate is a bit ugly, and sponsors will probably change a lot. You can also remove their logos etc